web/impact/impact/v1/helpers/model_helper.py

```diff
@@ -189,10 +198,8 @@ def save(self):
     def all_objects(cls):
         return cls.model.objects.all()
 
-    def field_name(self, field):
-        obj = getattr(self.subject, field, None)
-        if hasattr(obj, "name"):
-            return obj.name
+    def get_field_value(self, field):
```

Comment:
This needs a different name.  "get_" should only be used to refer explicitly to things related to "GET" calls (e.g., get_description in profile_helper.py).